src/components/CommentInfo/CommentInfo.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+type Props = {
+  comments: Comment;
+};
+
+export const CommentInfo: React.FC <Props> = (props) => {
+  const { comments } = props;
```

Comment:
```suggestion
  const { comment } = props;
```
Тут в нас один об'єкт з коментарем, а не декілька коментів